action.yml

```diff
@@ -20,6 +20,11 @@ inputs:
     description: Time to start application
     required: true
     default: 10
+  container-name:
+    description: Name of the container in case of "docker compose" command
+  build-delay:
+    description: Time to wait for docker container build
+    default: 30
```

Comment:
this will result to syntax errors as no unit is mentioned! you can change it to `50s`

Reply:
Ok

Reply:
should i also add unit in delay too i.e, ```10s```